old-simulator-doc.txt

```diff
@@ -68,21 +68,21 @@ foeModifyMove(user, target, move) [on target]
 
 	examples: [ability] Adaptability, [ability, foe ability] No Guard
 
-modifyPokemon(pokemon) [on pokemon]
-	Any effect that modifies a pokemon should be applied here.
+modifyPokémon(Pokémon) [on Pokémon]
```

Comment:
Function names probably shouldn't be changed.

Reply:
Oops sorry. I will fix that in a bit

Reply:
I fixed the function names, but accidentally deleted some of the code on lines 71 and 72. I will see if I can retrieve it.